Source/AliveLib/Input.hpp

```diff
@@ -0,0 +1 @@
+void __cdecl InputGetJoystickInput_460280(float *X1, float *Y1, float *X2, float *Y2, DWORD *Buttons);
```

Comment:
__cdecl has a define of CC for calling convention so it can easily be compiled out if need be